From 1b04d1a81a694332c4172c882fbef653c56b8af2 Mon Sep 17 00:00:00 2001
From: Neil Armstrong <narmstrong@baylibre.com>
Date: Fri, 16 Apr 2021 14:52:06 +0200
Subject: [PATCH 27/40] FROMLIST(v1): net: designware: meson8b: add g12a
 compatible

Add support for the Meson G12A dwmac glue compatible needed after Linux 5.12 sync.

Signed-off-by: Neil Armstrong <narmstrong@baylibre.com>
---
 drivers/net/dwmac_meson8b.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/net/dwmac_meson8b.c b/drivers/net/dwmac_meson8b.c
index c0b6ef4994..069bf1724d 100644
--- a/drivers/net/dwmac_meson8b.c
+++ b/drivers/net/dwmac_meson8b.c
@@ -133,6 +133,7 @@ static int dwmac_meson8b_probe(struct udevice *dev)
 
 static const struct udevice_id dwmac_meson8b_ids[] = {
 	{ .compatible = "amlogic,meson-gxbb-dwmac", .data = (ulong)dwmac_setup_gx },
+	{ .compatible = "amlogic,meson-g12a-dwmac", .data = (ulong)dwmac_setup_axg },
 	{ .compatible = "amlogic,meson-axg-dwmac", .data = (ulong)dwmac_setup_axg },
 	{ }
 };
-- 
2.17.1

